src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+    >
+      <ec-icon
+        class="ec-summary-info__icon"
+        data-test="ec-summary-info__icon"
+        :class="{'tw-mr-12': iconName}"
```

Comment:
Why we use here iconName as condition as we already check it on line 7?

Reply:
you are right, corrected